common/persistence/execution_manager_test.go

```diff
@@ -24,8 +24,13 @@ package persistence
 
 import (
 	"context"
+	"fmt"
+	"github.com/uber/cadence/common"
+	"github.com/uber/cadence/common/checksum"
+	"github.com/uber/cadence/common/types"
```

Comment:
make fmt

Reply:
ohwell. I've tried adding gci as file watcher, but it makes code writing very weird. I'll try to figure out how to set a helper for myself. Or maybe a recommit hook.